docs/examples/nonlocal_games/example_extended_nonlocal_games.py

```diff
@@ -711,6 +711,145 @@
 #
 # For further information on the :math:`G_{MUB}` game, consult :footcite:`Russo_2017_Extended`.
 
```

Comment:
this tutorial is not visible on the gallery page
https://toqito--1303.org.readthedocs.build/en/1303/auto_examples/index.html